openmc/cmfd.py

```diff
@@ -12,12 +12,65 @@
 
 from collections.abc import Iterable
 from numbers import Real, Integral
-from xml.etree import ElementTree as ET
 import sys
-
```

Comment:
Keep a blank line between standard-library imports and third-party library imports